.circleci/config.yml

```diff
@@ -167,17 +169,42 @@ jobs:
       XC_ARCH: "amd64"
 
   # build all arm/arm64 architecture supported OS binaries
-  build-arm-arm64:
-    <<: *build-distros
+  build-arm:
+    docker:
+      - image: *GOLANG_IMAGE
     environment:
-      <<: *build-env
-      XC_OS: linux
-      XC_ARCH: "arm arm64"
+      <<: *ENVIRONMENT
+      CGO_ENABLED: 1
+      GOOS: linux
+    steps:
+      - checkout
+      - run: sudo apt-get update && sudo apt-get install -y gcc-arm-linux-gnueabi gcc-arm-linux-gnueabihf gcc-aarch64-linux-gnu
```

Comment:
This step would be cached right?

Can we not control the container used to compile these. Like instead of using the circleci container maybe create our own with the necessary go tools and packages installed?

Reply:
This step only takes 10 seconds or so, so I'm fine with not having to manage our own containers for this purpose. Later on down the line if we have more drift from base `circleci/golang` I think we can consider pushing up and using our own containers.

Reply:
They must have an cache of the ubuntu repositories for it to be that quick. In that case this is fine.

Reply:
If this ever does get too slow, we could possibly leverage CircleCI's caching layer instead of maintaining our own Docker image https://discuss.circleci.com/t/caching-apt-get-packages/9076